includes/class-rest.php

```
@@ -351,6 +351,13 @@ public function optimise_image( \WP_REST_Request $request ) {
 			$webp_images = isset( $params['webp'] ) ? array_map( 'sanitize_text_field', (array) $params['webp'] ) : array();
 			$avif_images = isset( $params['avif'] ) ? array_map( 'sanitize_text_field', (array) $params['avif'] ) : array();
 
+			// If no paths sent from client, fall back to reading pending paths from DB.
+			if ( empty( $webp_images ) && empty( $avif_images ) ) {
+				$img_info    = Img_Converter::get_img_info();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'PerformanceOptimise\Inc\Img_Converter' in method 'optimise_image'. (undefined)

(StaticAccess)